RocketChat/Rocket.Chat.ReactNative
# [FIX] Roles rendering on dark theme

## Proposed changes

Roles not rendering properly on dark theme

## Issue(s)	


## How to test or reproduce

- Make sure you're logged in user with admin (or correct permission)
- Go to a room, and click on the avatar of another user that talked before
- You will be moved directly to RoomInfoView and you can see the user's roles

## Screenshots
<table>
<thead>
<tr>
<th>Before</th>
</thead>
</tr>
<tbody>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/47038980/148426084-147719f1-1093-4554-b42f-b6c59c531d18.png" height="450" />
<img src="https://user-images.githubusercontent.com/47038980/148426153-0c20e554-87dc-4715-b98a-52c53345dea0.png" height="450" />
<img src="https://user-images.githubusercontent.com/47038980/148426178-9b82e138-c6ae-4aa2-ac39-66f11cf1c9c8.png" height="450" />
</td>
</tr>
</tbody>
<thead>
<tr>
<th>After</th>
</thead>
</tr>
<tbody>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/47038980/148426474-a4c66222-2549-43c3-a5af-79683ce45c75.png" height="450" />
<img src="https://user-images.githubusercontent.com/47038980/148426511-709d145b-d794-477d-a90d-5c88255f0146.png" height="450" />
<img src="https://user-images.githubusercontent.com/47038980/148426545-282cf6e1-a0b3-42eb-9485-7d5ff0599239.png" height="450" />
</td>
</tr>
</tbody>
</table>

## Types of changes




## Checklist



## Further comments
